src/client/dfuse/SConscript

```diff
@@ -140,7 +140,7 @@ def check_ioctl_def(context, ctype):
     context.Message(f'Checking if fuse ioctl is type {ctype} ')
 
     # pylint: disable-next=consider-using-f-string
-    src = """#include <fuse3/fuse_lowlevel.h>
+    src = """#include <fused/fuse_lowlevel.h>
```

Comment:
This is for working with various fuse versions, we should replace this with a simple version check.